### ISSUE TITLE
Parsing string-value "Yes" has assertion-error

### ISSUE THREAD

**Issue by ErikTrapman** [yaz/yaz_messaging_plugin#2]
E.g.

```yaml
list:
    is_public_visible:
        not.public: No
        yes:
            from: "Yes, from: !date"
            till: "Yes, till !date"
            between: "Yes, between !date_from and !date_till"
            not_scheduled: "Yes"
```
returns
`AssertionError: ['admin.nl.yml', '.list.is_public_visible', True, OrderedDict([('from', 'Yes, from: !date'), ('till', 'Yes, till !date'), ('between', 'Yes, between !date_from and !date_till'), ('not_scheduled', 'Yes')])]`

**Comment by boudewijn-zicht** [comment]
In this case the problem is the `yes:` key, not any of the quoted values.  Although the non-quoted `No` may also cause problems.  But one issue at a time.

Yaml defines the following types for booleans, these, and other reserved words, must be quoted when using in yaml: http://yaml.org/type/bool.html

**Comment by boudewijn-zicht** [comment]
Parsing of types defined in YAML 1.1 have been disabled.  Fixed in version 1.0.3
